frontend/src/layouts/MainLayout.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <v-app>
+    <v-main>
+      <router-view/>
+      <toast-layer/>
+    </v-main>
+  </v-app>
+</template>
+
```

Comment:
layouts is fine - ideally leave this folder under src- and add it to the Readme.
I optimized on view and thre layouts dir out :-) but i was also in doubt - and like oyu brought it back